src/Microsoft.Diagnostics.Monitoring.Options/AzureBlobEgressProviderOptions.cs

```diff
@@ -85,7 +85,7 @@ internal sealed partial class AzureBlobEgressProviderOptions :
         [Display(
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AzureBlobEgressProviderOptions_Metadata))]
-        public IDictionary<string, string> Metadata { get; }
+        public IDictionary<string, string> Metadata { get; set; }
```

Comment:
Do you want to add comments regarding the setter?

Reply:
The issue is going to be fixed in RC 2, so I don't think it is worthwhile to describe a situation that will be fixed soon.